lib/cdo/contact_rollups/v2/contact_rollups.rb

```diff
@@ -18,14 +18,16 @@ def self.build_contact_rollups(log_collector)
       ContactRollupsComparison.compile_processed_data
     end
 
-    log_collector.time!('Downloads new email-Pardot ID mappings') do
-      ContactRollupsPardotMemory.add_and_update_pardot_ids
-    end
-    log_collector.time!('Creates new Pardot prospects') do
-      ContactRollupsPardotMemory.create_new_pardot_prospects
-    end
-    log_collector.time!('Updates existing Pardot prospects') do
-      ContactRollupsPardotMemory.update_pardot_prospects
+    if pardot_steps
```

Comment:
I think this belongs to another PR when we want to turn on the pipeline (with new cronjob entry). However, this is a 2-line change so it's okay if you want to keep it here.
```suggestion
if sync_with_pardot
```